### DISCUSSION TITLE
Does the stats/leaderbord only show to player if they played once?

### DISCUSSION THREAD

**Discussion by HappyAreaBean** [diamond-rip/Eden#51] [Q&A] [answered]
Do the statistics/leaderboard only show to players if they have played once?
![圖片](https://user-images.githubusercontent.com/50882051/210791131-d40b953d-f500-4b34-8801-6e5cbad26fac.png)

**Comment by GoodestEnglish** [comment, marked as answer]
Yes. They have to be logged in at least once to allow server to search their statistics
